pkg/api/types.go

```diff
@@ -271,8 +271,16 @@ type KubernetesAddon struct {
 	Data       string                    `json:"data,omitempty"`
 }
 
-// IsEnabled returns if the addon is explicitly enabled, or the user-provided default if non explicitly enabled
-func (a *KubernetesAddon) IsEnabled(ifNil bool) bool {
+// IsEnabled returns if the addon is enabled
+func (a *KubernetesAddon) IsEnabled() bool {
```

Comment:
this func doesn't have UT coverage 

https://codecov.io/gh/Azure/aks-engine/pull/1409/src/pkg/api/types.go